extensions/omni_httpc/omni_httpc.c

```diff
@@ -557,7 +640,7 @@ Datum http_execute(PG_FUNCTION_ARGS) {
   sockpool->capacity = Max(num_requests, sockpool->capacity);
 
   // Load CA bundle if necessary
-  if (ssl_targets && sockpool->_ssl_ctx == NULL) {
+  if (sockpool->_ssl_ctx == NULL) {
```

Comment:
The most common redirects are HTTP -> HTTPS, so setting up the SSL context conditionally cannot be done any longer. AFAIK Nowadays SSL is pretty much the default anyways.